docs/cloud/http-api.md

```diff
@@ -362,7 +362,7 @@ Graphite-web-like api. It can return JSON, pickle or messagepack output
 * target: mandatory. one or more metric names or [patterns](#graphite-patterns).
 * from: see [timespec format](#tspec) (default: 24h ago) (exclusive)
 * to/until : see [timespec format](#tspec)(default: now) (inclusive)
-* format: json, msgp, pickle, or msgpack (default: json)
+* format: json, msgp, pickle, or msgpack (default: json) (note: msgp and msgpack are similar, but msgpack is for use with graphite)
```

Comment:
I feel like this doesn't do a great job of clarifying... Is this saying we shouldn't use msgpack because that's only intended to be used by/for graphite?

Reply:
if you want to use messagepack for graphite, you should use 'msgpack', not 'msgp'